patches/cloud_regress_pg16.patch

```diff
@@ -0,0 +1,3949 @@
+diff --git a/src/test/regress/expected/aggregates.out b/src/test/regress/expected/aggregates.out
+index 0c24f6afe4..dd808ac2b4 100644
+--- a/src/test/regress/expected/aggregates.out
++++ b/src/test/regress/expected/aggregates.out
+@@ -11,7 +11,8 @@ CREATE TABLE aggtest (
+ 	b			float4
+ );
+ \set filename :abs_srcdir '/data/agg.data'
+-COPY aggtest FROM :'filename';
++\set command '\\copy aggtest FROM ' :'filename';
++:command
```

Comment:
Q: Why this change was needed?

Reply:
We are running the regression test against the cloud instance (staging).
This instance does not have the files, which are available for pg_regress. That's why we changed the server-side copy to the client-side.